agents/che-core-api-agent/src/main/java/org/eclipse/che/api/agent/server/launcher/AbstractAgentLauncher.java

```diff
@@ -66,13 +68,33 @@ public AbstractAgentLauncher(long agentMaxStartTimeMs,
         this.agentLaunchingChecker = agentLaunchingChecker;
     }
 
+    /**
+     * Launches agent script in machine and waits until agent starts.
+     * The machine should be started.
+     * Retrieves agent's logs if its start fails.
```

Comment:
As far as I can see it logs agent logs if it fails, but stores it in any case. Is it needed to add logging related logic into javadocs?